src/Enums/UserRole.tsx

```diff
@@ -55,10 +60,11 @@ export const getUserRole = (): UserRole => {
 };
 
 export const getUserId = () => {
-    const userId = Cookies.get(CookieEnum.USERID);
+    // const userId = Cookies.get(CookieEnum.USERID);
+    const userId = session.userId;
     let userIdValue = 0; 
 
-    if (userId === undefined || isNaN(userIdValue = parseInt(userId, 10))) {
+    if (userId === null || isNaN(userIdValue = parseInt(userId, 10))) {
```

Comment:
Unrelated, but I don't see any reason for the parseInt to be in the conditional here.

Reply:
Yeah, when I clean up I will move this into the local storage fetch like I do elsewhere